rust/parquet_derive/src/parquet_field.rs

```diff
@@ -252,6 +252,8 @@ impl Field {
     }
 }
 
+#[allow(clippy::enum_variant_names)]
```

Comment:
While clippy has some valid points with these warnings, I did not want to address them in this PR